scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1455,8 +1455,10 @@ bool antistall_consume(struct cpu_ctx *cpuc)
 	bpf_for_each(scx_dsq, p, *antistall_dsq, 0) {
 		cur_delay = get_delay_sec(p, jiffies_now);
 
-		if (cur_delay > antistall_sec)
+		if (cur_delay > antistall_sec) {
+			gstat_inc(GSTAT_ASTALL, cpuc);
```

Comment:
Can you name it `GSTAT_ANTISTALL`? It's not that long to begin with. Also, shouldn't this count each consumption event? ie. whenever `scx_bpf_dsq_move_to_local()` returns `true`?